Automattic/wp-calypso
# Upgrade wordpress monorepo

#### Proposed Changes
Upgrade WordPress monorepo to the latest version:

- Add the `upgrade-interactive` yarn plugin. (This accounts for 541 lines of the diff -- so we're really only adding around 300 lines in this PR, not 800.)
- Ran that to upgrade every WordPress and @types/wordpress package.
- Fixed peer dependency resolutions in yarnrc.yml.
- Deduped dependencies.
- TBD to fix errors that arise

#### Testing Instructions
1. Smoke test calypso.live, especially any WordPress components. Test components in devdocs as well.
2. Verify local yarn install and yarn build continue to work as expected.
3. Smoke test ETK.
